### PR TITLE
[UI]Resolved Unhandled Runtime Error on opening kubernates nav and Any button together

### LINKED ISSUE #5306: [UI]Unhandled Runtime Error on opening kubernates nav and any button together
#### Current Behavior

Unhandled Runtime Error on opening kubernates nav and slider button together

https://user-images.githubusercontent.com/56961488/160894639-e3942239-462c-4c3e-9701-01f97eed6077.mp4

#### Desired Behavior

No Error like this should occur while opening these navs together
#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Harshit-Dandriyal**
Signed-off-by: Harshit-Dandriyal <dandriyalh9318@gmail.com>

**Description**
Unhandled Runtime Error on opening kubernates nav and Any button together...

Before ---

https://user-images.githubusercontent.com/56961488/161300321-eb77e3a2-feca-4768-948f-0c2d63f1ab49.mp4
 
After this PR --

https://user-images.githubusercontent.com/56961488/161300442-c64147f6-ced1-4050-b60a-03df6ea186e5.mp4

This PR fixes #5306

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Harshit-Dandriyal** [comment]
@warunicorn19 I think Optional Chaining was not a issue with this, Our e.target.className.include is actually not a function anymore(may be removed with a update in chrome browser), As we can see
 
Screenshot 2022-04-01 214309

e.target.classname has only few classes....

had a great time with this small UI Bug😂

**Comment by Harshit-Dandriyal** [comment]
@warunicorn19 can you please review this?

**Review by adithyaakrishna — DISMISSED** [review]
LGTM 🎉

File: ui/components/Header.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ function K8sContextMenu({
       <Slide direction="down" timeout={300} in={open} style={{ position : "absolute", left : "-5rem", zIndex : "-1", bottom : "-75%", transform : showFullContextMenu ? `translateY(${transformProperty}%)`: "translateY(0)" }} mountOnEnter unmountOnExit>
         <div>
           <ClickAwayListener onClickAway={(e) => {
-            if (!e.target.className.includes("cbadge") && e.target.className !="k8s-image" && !e.target.className.includes("k8s-icon-button")) {
+            if (!e.target.classList.contains("cbadge") && e.target.className !="k8s-image" && !e.target.classList.contains("k8s-icon-button"))  {
```
Comment (asubedy) [inline ui/components/Header.js:217]: @Harshit-Dandriyal did you confirm that `className` does not have the `cbadge` and `k8s-con-button`

Comment (Harshit-Dandriyal) [reply ui/components/Header.js:217]: I posted a screenshot above where all the classes in classname is there, I think cbadge is not there

**Review by asubedy — APPROVED** [review]
LGTM

**Comment by Harshit-Dandriyal** [comment]
@leecalcote It was issue #5310 and Yes, I am sorry, offline Exams are going, also Some freelance projects were remaining, After Tomorrow, I will solve all the pending issues :)

**Comment by leecalcote** [comment]
@Harshit-Dandriyal That’s fine, just don’t ask for the PR to be merged without having incorporated all the feedback given.

**Comment by Abhishek-kumar09** [comment]
@Harshit-Dandriyal classList and className is similar thing in different format. Can you confirm that if any of the k8s related className is actually applied to `e.target`. From your screenshot above I can only see the Mui classes applied and no others. May I know when some of the rest of classes written in if statement is applied.

**Comment by Harshit-Dandriyal** [comment]
> @Harshit-Dandriyal classList and className is similar thing in different format. Can you confirm that if any of the k8s related className is actually applied to `e.target`. From your screenshot above I can only see the Mui classes applied and no others. May I know when some of the cases where rest of classes written in `if` statement is applied. 
> 

Actually It is screenshot of console.log(e.target), I see there are only mui classes appearing in classname

**Comment by Harshit-Dandriyal** [comment]
@Abhishek-kumar09 Review Please

**Comment by adithyaakrishna** [comment]
Test Comment

**Comment by leecalcote** [comment]
@Harshit-Dandriyal what would you like to do with this PR?

**Comment by Harshit-Dandriyal** [comment]
> @Harshit-Dandriyal what would you like to do with this PR?

oh yeah, This PR is ready to merge, This is still an issue causing problems :smile:

**Comment by Harshit-Dandriyal** [comment]
@leecalcote Actually, This PR is having a lot of merge conflicts and wrong formatting, Creating New PR.

**Pull request closed without merging**